third_party/openmetadata/Makefile

```diff
@@ -6,8 +6,18 @@ deploy-openmetadata:
 	helm repo add open-metadata https://helm.open-metadata.org/ || true
 	helm repo update || true
 	kubectl create ns ${INSTALLATION_NAMESPACE} || true
-	sed s/INSTALLATION_NAMESPACE/${INSTALLATION_NAMESPACE}/g pv1.yaml | kubectl apply -n ${INSTALLATION_NAMESPACE} -f - || true
-	sed s/INSTALLATION_NAMESPACE/${INSTALLATION_NAMESPACE}/g pv2.yaml | kubectl apply -n ${INSTALLATION_NAMESPACE} -f - || true
+ifeq ($(OPENSHIFT_INSTALLATION), true)
+	oc adm policy add-scc-to-user anyuid -z airflow -n ${INSTALLATION_NAMESPACE}
+	oc adm policy add-scc-to-user anyuid -z mysql -n ${INSTALLATION_NAMESPACE}
+	oc adm policy add-scc-to-user ibm-privileged-scc -z default -n ${INSTALLATION_NAMESPACE}
+	kubectl apply -f pvc3.yaml -n ${INSTALLATION_NAMESPACE} || true
```

Comment:
should it be `kubectl apply -f openshift/pvc3.yaml -n ${INSTALLATION_NAMESPACE} || true` ?

Reply:
yes

Reply:
The name in the temp directory is `pvc3.yaml`, not `openshift/pvc3.yaml`